### DISCUSSION TITLE
JSON Schema

### DISCUSSION THREAD

**Discussion by lentex** [mateusjunges/laravel-kafka#265] [Ideas]
Hi @mateusjunges,
first things first: thank you very much for providing such a cool, clean package for interacting with Kafka in Laravel projects 👏 

One feature I'm missing right now is the support for JSON Schema ("only" support for Avro right now).

So my question is if there is a plan to support this in the future (v2)?

I'm afraid that I can't provide a PR for this kind of thing as I'm not deep enough into this whole topic.

Thanks in advance and keep up the excellent work 💪

**Comment by mateusjunges** [comment]
Hey 👋 thanks for the kind words! I currently don't have plans to implement the json schema. I can research the topic as I haven't used json scheam myself, but I don't think it will be available with the first release of v2.
